### DISCUSSION TITLE
Improve translation

### DISCUSSION THREAD

**Discussion by AP2020C** [Keriew/augustus#598] [General]
I would simply ask to add the Italian translation of the manual.
I would also like to point out that some texts in the game are not translated, they should be improved or they overlap.

**Comment by Ouaz** [comment]
For the game translation in italian, do you know how to open a PR in Github?
Some Augustus Team members are italian, and could review your changes/fixes.

**Reply by berseker** [reply]
> For the game translation in italian, do you know how to open a PR in Github? Some Augustus Team members are italian, and could review your changes/fixes.

I am also interested in helpng improving Italian translation (I am italian). by PR you mean Pull Request? I can open that if needed

**Reply by Ouaz** [reply]
Yes, just go here: https://github.com/Keriew/augustus/blob/master/src/translation/italian.c

Click the Pen icon in the upper-right corner, edit what you want to translate/improve/fix, then click "Propose Changes" at the bottom.
It will automatically create a fork of Augustus in your profile (here: https://github.com/berseker?tab=repositories). 

On the next screen, simply click "Create a Pull Request".
